src/library_memfs.js

```diff
@@ -76,7 +76,7 @@ addToLibrary({
       if (FS.isDir(node.mode)) {
         node.node_ops = MEMFS.ops_table.dir.node;
         node.stream_ops = MEMFS.ops_table.dir.stream;
-        node.contents = {};
+        node.contents = new Map();
```

Comment:
Is this object never used by `library_fs.js`?

Reply:
There is a `.contents` field there as well, which makes it hard to do a quick search.